tests/includes/wait-for.sh

```diff
@@ -52,7 +52,8 @@ idle_subordinate_condition() {
     unit_index=${4:-0}
     parent_index=${5:-0}
 
-    echo ".applications | select(.[\"$parent\"] | .units | .[\"$parent/$parent_index\"] | .subordinates | .[\"$name/$unit_index\"] | .[\"juju-status\"] | .current == \"idle\") | keys[$app_index]"
+    # Print the *subordinate* name if it has an idle status in parent application
```

Comment:
@SimonRichardson I think the original condition was wrong because it would return the name of the parent instead of the subordinate. wdyt?